libbeat/metric/system/host/host.go

```diff
@@ -72,6 +73,7 @@ func ReportInfo(_ monitoring.Mode, V monitoring.Visitor) {
 		monitoring.ReportString(V, "platform", info.OS.Platform)
 		monitoring.ReportString(V, "version", info.OS.Version)
 		monitoring.ReportString(V, "family", info.OS.Family)
+		monitoring.ReportString(V, "kernel", info.KernelVersion)
```

Comment:
@ycombinator FYI: This will mean it will also be reported to Monitoring. I don't think we need to add it to the Elasticsearch template but it's good to know it's there.